playwright-e2e/tests/dsm/osteo/os1-participant-p4a42b.spec.ts

```diff
@@ -36,6 +36,8 @@ test.describe.serial('Same Participant in Osteo1 and Osteo2', () => {
       expect(await participantPage.getRegistrationDate()).toContain('Aug 21, 2020');
       expect(await participantPage.getShortId()).toStrictEqual(shortID);
       expect(await participantPage.getGuid()).toStrictEqual('S4TF6YG65972AJV775LA');
+      await page.evaluate(() => { window.scrollTo(0, 0); });
```

Comment:
not sure if this will fix it but it was recommended - and passes locally - so we'll see